modules/overviews/app/components/overviews/widgets/members_component.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+module Overviews
+  module Widgets
+    class MembersComponent < Grids::WidgetComponent
+      MEMBERS_LIMIT = 5
+
+      attr_reader :project, :current_user
+
+      def initialize(project)
+        super()
+
+        @project = project
+        @current_user = current_user
```

Comment:
This method signature will break handling of `current_user`.

I appreciate that this is not (yet) properly documented, but `Grids::WidgetComponent` uses `dry-initializer` to generate an initializer and accessor methods.

Therefore, the following should suffice.

```suggestion
      param :project
      # option :current_user is defined in parent class

      def initialize(*)
        super
```